includes/admin/views/review.php

```diff
@@ -0,0 +1,70 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'You are not allowed to call this page directly.' );
+}
+?>
+<div class="notice notice-info is-dismissible wpbdp-review-notice">
+	<div class="wpbdp-satisfied">
+		<p>
+			<?php echo esc_html( $title ); ?>
+			<br/>
+			<?php esc_html_e( 'Are you enjoying Business Directory Plugin?', 'business-directory-plugin' ); ?>
+		</p>
+		<a href="#" class="wpbdp_reverse_button wpbdp_animate_bg show-wpbdp-feedback wpbdp-button-secondary" data-link="feedback">
```

Comment:
We're missing the button styling here. Here's what it looks like in Formidable:
<img width="469" alt="Screen Shot 2021-10-11 at 5 43 40 PM" src="https://user-images.githubusercontent.com/1116876/136867858-0bd3329e-3936-432b-9c2d-6ac90a97f720.png">
<img width="472" alt="Screen Shot 2021-10-11 at 5 43 30 PM" src="https://user-images.githubusercontent.com/1116876/136867864-412a0bcd-56db-4b52-aef9-ec2c755c0db9.png">
<img width="477" alt="Screen Shot 2021-10-11 at 5 43 23 PM" src="https://user-images.githubusercontent.com/1116876/136867867-32225d20-0b43-414b-81f6-83de1b6534dd.png">